site/src/components/pages/home/Header/HeaderCardSvg.tsx

```diff
@@ -0,0 +1,720 @@
+            result="hardAlpha"
+          />
+          <feOffset dy={0.5} />
+          <feGaussianBlur stdDeviation={0.5} />
+          <feColorMatrix
+            type="matrix"
+            values="0 0 0 0 0.152941 0 0 0 0 0.196078 0 0 0 0 0.337255 0 0 0 0.15 0"
+          />
+          <feBlend
+            mode="multiply"
+            in2="BackgroundImageFix"
+            result="effect1_dropShadow_2132_16425"
+          />
+          <feColorMatrix
+            in="SourceAlpha"
+            type="matrix"
+            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+            result="hardAlpha"
+          />
+          <feOffset dy={2.59259} />
+          <feGaussianBlur stdDeviation={3.22106} />
+          <feColorMatrix
+            type="matrix"
+            values="0 0 0 0 0.152941 0 0 0 0 0.196078 0 0 0 0 0.337255 0 0 0 0.08 0"
+          />
+          <feBlend
+            mode="normal"
+            in2="effect1_dropShadow_2132_16425"
+            result="effect2_dropShadow_2132_16425"
+          />
+          <feColorMatrix
+            in="SourceAlpha"
+            type="matrix"
+            values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0"
+            result="hardAlpha"
+          />
+          <feOffset dy={4} />
+          <feGaussianBlur stdDeviation={5.5} />
+          <feColorMatrix
+            type="matrix"
+            values="0 0 0 0 0.152941 0 0 0 0 0.196078 0 0 0 0 0.337255 0 0 0 0.04 0"
+          />
+          <feBlend
+            mode="normal"
+            in2="effect2_dropShadow_2132_16425"
+            result="effect3_dropShadow_2132_16425"
+          />
+          <feBlend
+            mode="normal"
+            in="SourceGraphic"
+            in2="effect3_dropShadow_2132_16425"
+            result="shape"
+          />
+        </filter>
+        <filter
+          id="filter6_f_2132_16425"
+          x={0}
+          y={413}
+          width={136}
+          height={136}
+          filterUnits="userSpaceOnUse"
+          colorInterpolationFilters="sRGB"
+        >
+          <feFlood floodOpacity={0} result="BackgroundImageFix" />
+          <feBlend
+            mode="normal"
+            in="SourceGraphic"
+            in2="BackgroundImageFix"
+            result="shape"
+          />
+          <feGaussianBlur
+            stdDeviation={2}
+            result="effect1_foregroundBlur_2132_16425"
+          />
+        </filter>
+        {!hideGradient && (
+          <linearGradient
+            id="paint0_linear_2132_16425"
+            x1={715.709}
+            y1={468}
+            x2={515.67}
+            y2={467.28}
+            gradientUnits="userSpaceOnUse"
+          >
+            <stop stopColor="#2A2B37" />
+            <stop offset={1} stopColor="#F7FAFC" stopOpacity={0} />
+          </linearGradient>
+        )}
+        <image
+          id="image0_2132_16425"
+          width={200}
+          height={200}
+          href="/assets/header-card-svg-photo.png"
```

Comment:
@CiaranMn the committed image is used here in svg

Reply:
Oops, thank you